src/autosave.js

```diff
@@ -0,0 +1,254 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module autosave/autosave
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import PendingActions from '@ckeditor/ckeditor5-core/src/pendingactions';
+import DomEmitterMixin from '@ckeditor/ckeditor5-utils/src/dom/emittermixin';
+import throttle from './throttle';
+
+/* globals window */
+
+/**
+ * Autosave plugin provides an easy-to-use API to save the editor's content.
+ * It watches {@link module:engine/model/document~Document#event:change:data change:data}
+ * and `Window:beforeunload` events and calls the {@link module:autosave/autosave~Adapter#save} method.
```

Comment:
window - lower case

Reply:
`window#beforeunload` actually. But `Window#beforeunload` is ok too – it's a type as well as variable :D